pkg/hostpath/hpv.go

```diff
@@ -77,6 +81,11 @@ func (hpv *hostPathVolume) GetSharedDataId() string {
 	defer hpv.Lock.Unlock()
 	return hpv.SharedDataId
 }
+func (hpv *hostPathVolume) GetSharedDataVersion() string {
+	hpv.Lock.Lock()
+	defer hpv.Lock.Unlock()
+	return hpv.ShareDataVersion
+}
```

Comment:
We should have a new line here, right? Have we run `go-fmt` against this file?

Reply:
yep no changes from gofmt:

```
gmontero ~/go/src/github.com/openshift/csi-driver-projected-resource  (master)$ make verify
go vet ./cmd/... ./pkg/... ./test/...
gofmt -w ./cmd/ ./pkg/ ./test/
gmontero ~/go/src/github.com/openshift/csi-driver-projected-resource  (master)$ git status 
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
gmontero ~/go/src/github.com/openshift/csi-driver-projected-resource  (master)$ 
```